src/os/redox.rs

```diff
@@ -0,0 +1,15 @@
+#![cfg(target_os = "redox")]
+
+use std::sync::{Arc, Mutex};
+
+use Window;
+
+pub trait WindowExt {
+    fn get_orbclient_window(&self) -> Arc<Mutex<orbclient::Window>>;
+}
+
+impl WindowExt for Window {
+    fn get_orbclient_window(&self) -> Arc<Mutex<orbclient::Window>> {
+        self.window.get_orbclient_window()
+    }
+}
```

Comment:
This means that `orbclient` becomes a public dependency of `winit`. Given `orbclient` is the official API of Redox (I assume it is at least?), I'd tend to be okay with that.

However this means that any breaking change to `orbclient` means a breaking release of `winit` to update it, which is problematic. For this reason `winit` has tried to reduce public dependencies to a minimum.

So, what is the stability story of `orbclient`? What do you think @Osspial @francesca64 ?

Reply:
I'm fine with this. There hasn't been an incompatible change in almost 2 years: https://crates.io/crates/orbclient/versions